docs/antora-playbook.yml

```diff
@@ -9,8 +9,8 @@ site:
 content:
   sources:
     - url: ./../
-      branches: [HEAD, 'v*']
-      edit_url: 'https://github.com/cerbos/cerbos/tree/{refname}/{path}'
+      branches: [HEAD, 'v*', '!v0.[0-25].*']
```

Comment:
Aren't the branches named e.g. `v0.25`, so the trailing `.*` should be omitted?

Reply:
Also, are you sure about this pattern syntax? I couldn't find it in the [docs](https://docs.antora.org/antora/latest/playbook/content-refname-matching/).

Reply:
Good point about the branch name 👍🏽 
I am following this: https://docs.antora.org/antora/latest/playbook/content-branches/#default. It seems to suggest that the pattern is a regex. It does work in the local build. But it could be just a lucky coincidence.

Reply:
I don't think so, I think it's an extended glob. I think we need `!v0.{0..25}`.

Reply:
Well, even `!v0.{0..25}` seems to work. I am not quite sure whether they are using globs or regexes 🤷🏽

Reply:
Their examples aren't valid regexps. The doc you're on links to the one I looked at:

>Antora also supports matching branch names using exclusions, braces, alternation, ranges, and repetition patterns. See Refname Matching in Content Sources.

Reply:
Yep, shouldn't have skimmed the docs in between calls 🙈